### PR TITLE
first pass for eos5

### PR DESCRIPTION
**Pull request by MichaelDrydenEndless**
Work in Progress

https://phabricator.endlessm.com/T33856

### CODE REVIEW

File: gnome-help/C/a11y-contrast.page [hunk 1]
```diff
@@ -41,8 +41,7 @@
 
   <steps>
     <item>
-      <p>Open the <gui xref="shell-introduction#activities">Activities</gui> overview and
-      start typing <gui>Accessibility</gui>.</p>
+      <p>Go to the Dekstop and start typing <gui>Accessibility</gui>.</p>
```
Comment (cassidyjames) [inline gnome-help/C/a11y-contrast.page:44]: Itty bitty typo

```suggestion
      <p>Go to the Desktop and start typing <gui>Accessibility</gui>.</p>
```

File: gnome-help/C/a11y-icon.page [hunk 2]
```diff
@@ -38,8 +38,7 @@
 
   <steps>
     <item>
-      <p>Open the <gui xref="shell-introduction#activities">Activities</gui> overview and
-      start typing <gui>Accessibility</gui>.</p>
+      <p>Go to the Dekstop and start typing <gui>Accessibility</gui>.</p>
```
Comment (cassidyjames) [inline gnome-help/C/a11y-icon.page:41]:
```suggestion
      <p>Go to the Desktop and start typing <gui>Accessibility</gui>.</p>
```

File: gnome-help/C/a11y-right-click.page [hunk 3]
```diff
@@ -45,8 +45,7 @@
 
   <steps>
     <item>
-      <p>Open the <gui xref="shell-introduction#activities">Activities</gui> 
-      overview and start typing <gui>Accessibility</gui>.</p>
+      <p>Go to the Dekstop and start typing <gui>Accessibility</gui>.</p>
```
Comment (cassidyjames) [inline gnome-help/C/a11y-right-click.page:48]:
```suggestion
      <p>Go to the Desktop and start typing <gui>Accessibility</gui>.</p>
```

File: gnome-help/C/a11y-braille.page [hunk 4]
```diff
@@ -27,11 +27,7 @@
 
   <title>Read screen in Braille</title>
 
-  <p>The <app>Orca</app> screen reader can display the user interface on a
-  refreshable Braille display. Depending on how you installed your system, you
-  might not have Orca installed. If not, install Orca first.</p>
-
-  <p if:test="action:install"><link style="button" action="install:orca">Install Orca</link></p>
+  <p>The <app>Orca</app> screen reader can speak the user interface.</p>
```
Comment (wjt) [inline gnome-help/C/a11y-braille.page:30]: I think it's OK to just leave this paragraph as-is. This section is about Braille displays, not audible screen-reading. And the "Depending on…" sentence is technically true, in the sense that Orca is always installed.

File: gnome-help/C/a11y-contrast.page (hunk 1 above)
Comment (wjt) [reply gnome-help/C/a11y-contrast.page:44]: I don't like the Germanic Noun Capitalization of Desktop. I suggest:

<gui>desktop</gui>

File: gnome-help/C/accounts-add.page [hunk 5]
```diff
@@ -69,7 +68,7 @@ desktop. Thus, your email client, calendar and other related
     </item>
     <item>
       <p>If you have entered your credentials correctly, you will be prompted
-      to allow GNOME access to your online account. Authorize access to
+      to allow Endless access to your online account. Authorize access to
```
Comment (wjt) [inline gnome-help/C/accounts-add.page:71]: Actually the prompt does ask you to allow GNOME access, so the original text is correct.

File: gnome-help/C/a11y-visualalert.page [hunk 6]
```diff
@@ -45,8 +50,7 @@
 
   <steps>
     <item>
-      <p>Open the <gui xref="shell-introduction#activities">Activities</gui> overview and
-      start typing <gui>Accessibility</gui>.</p>
+      <p>Go tp the desktop and start typing <gui>Accessibility</gui>.</p>
```
Comment (cassidyjames) [inline gnome-help/C/a11y-visualalert.page:53]:
```suggestion
      <p>Go to the desktop and start typing <gui>Accessibility</gui>.</p>
```

File: gnome-help/C/a11y-locate-pointer.page.stub [hunk 7]
```diff
@@ -27,7 +27,7 @@
 
   <steps>
     <item>
-      <p>Click your name on the top bar and select <gui>Settings</gui>.</p>
+      <p>Click the Battery Icon on the top bar and select <gui>Settings</gui>.</p>
```
Comment (cassidyjames) [inline gnome-help/C/a11y-locate-pointer.page.stub:30]:
```suggestion
      <p>Click the battery icon on the top bar and select <gui>Settings</gui>.</p>
```

File: gnome-help/C/accounts-remove.page [hunk 8]
```diff
@@ -41,11 +46,11 @@
   <p>You can remove an online account which you no longer want to use.</p>
 
   <note style="tip">
-    <p>Many online services provide an authorization token which your desktop stores
+    <p>Many online services provide an authorization token which your Desktop stores
     instead of your password. If you remove an account, you should also revoke
     that certificate in the online service. This will ensure that no other
     application or website can connect to that service using the authorization
-    for your desktop.</p>
+    for your Desktop.</p>
```
Comment (cassidyjames) [inline gnome-help/C/accounts-remove.page:53]:
```suggestion
    for your desktop.</p>
```

File: gnome-help/C/accounts-remove.page [hunk 9]
```diff
@@ -41,11 +46,11 @@
   <p>You can remove an online account which you no longer want to use.</p>
 
   <note style="tip">
-    <p>Many online services provide an authorization token which your desktop stores
+    <p>Many online services provide an authorization token which your Desktop stores
```
Comment (cassidyjames) [inline gnome-help/C/accounts-remove.page:49]:
```suggestion
    <p>Many online services provide an authorization token which your desktop stores
```

File: gnome-help/C/shell-windows.page [hunk 10]
```diff
@@ -21,9 +26,8 @@
 
   <title>Windows and workspaces</title>
 
-  <p>Like other desktops, the system uses windows to display your running
-  applications. Using both the <gui xref="shell-introduction#activities">Activities</gui>
-  overview and the <em>dash</em>, you can launch new applications and control
+  <p>Like other <gui>desktop</gui>, the system uses windows to display your running
+  applications. Using both the gui>desktop</gui> and the <em>dash</em>, you can launch new applications and control
```
Comment (wjt) [inline gnome-help/C/shell-windows.page:30]:
```
Error: Could not parse document:
 ..//C/shell-windows.page:30:  parser  error :  Opening and ending tag mismatch: p line 29 and gui
   applications. Using both the gui>desktop</gui> and the <em>dash</em>, you can 
```

```suggestion
  applications. Using both the <gui>desktop</gui> and the <em>dash</em>, you can launch new applications and control
```

File: gnome-help/C/sound-nosound.page [hunk 11]
```diff
@@ -86,8 +90,7 @@
 
   <steps>
     <item>
-      <p>Open the <gui xref="shell-introduction#activities">Activities</gui>
-      overview and start typing <gui>Sound</gui>.</p>
+      <p>Go to the gui>desktop</gui> and start typing <gui>Sound</gui>.</p>
```
Comment (wjt) [inline gnome-help/C/sound-nosound.page:93]: Same error here

File: gnome-help/C/tips-specialchars.page [hunk 12]
```diff
@@ -71,61 +76,6 @@
     </steps>
   </section>
 
-  <section id="compose">
```
Comment (wjt) [inline gnome-help/C/tips-specialchars.page:74]: Why are we removing this section?

File: gnome-help/C/user-changepicture.page [hunk 13]
```diff
@@ -35,16 +40,14 @@
 
   <p>When you log in or switch users, you will see a list of users with
   their login photos. You can change your photo to a stock image or an
-  image of your own. You can even take a new login photo with your
-  webcam.</p>
```
Comment (wjt) [inline gnome-help/C/user-changepicture.page:39]: Why are we removing this sentence? It's true: you can take a login photo with your webcam.

File: gnome-help/C/a11y-visualalert.page [hunk 14]
```diff
@@ -45,8 +50,7 @@
 
   <steps>
     <item>
-      <p>Open the <gui xref="shell-introduction#activities">Activities</gui> overview and
-      start typing <gui>Accessibility</gui>.</p>
+      <p>Go tp the <gui>desktop</gui> and start typing <gui>Accessibility</gui>.</p>
```
Comment (wjt) [inline gnome-help/C/a11y-visualalert.page:53]:
```suggestion
      <p>Go to the <gui>desktop</gui> and start typing <gui>Accessibility</gui>.</p>
```

File: gnome-help/C/sharing-desktop.page [hunk 15]
```diff
@@ -41,7 +45,7 @@
     <p>Networks section added without explanatory details.</p>
   </comment>
 
-  <p>You can let other people view and control your desktop from another
+  <p>You can let other people view and control your <gui>desktop</gui> from another
```
Comment (wjt) [inline gnome-help/C/sharing-desktop.page:48]: I don't think we should make this change and the similar changes in this file, for a sad technical reason: adding this `<gui>...</gui>` tag will invalidate all translations of this entire paragraph and make us/our community either retranslate it, or show the paragraph in English.

File: gnome-help/C/shell-apps-open.page [hunk 16]
```diff
@@ -38,19 +43,18 @@ to overview in list item below, not just workspace thumbnail.) Right-click
...
 
-  <p if:test="!platform:gnome-classic">Move your mouse pointer to the
-  <gui>Activities</gui> corner at the top left of the screen to show the
-  <gui xref="shell-introduction#activities">Activities</gui> overview. This is where you
-  can find all of your applications. You can also open the overview by pressing
-  the <key xref="keyboard-key-super">Super</key> key.</p>
+  <p if:test="!platform:gnome-classic">If you are not on the <gui>desktop</gui>, select 
+  the <gui>Applications</gui> button on the left side of the top bar to return 
+  to the <gui>desktop</gui>. This is where you can find all of your applications. You can 
+  also open the <gui>desktop</gui> by pressing the <key xref="keyboard-key-super">Super</key>
```
Comment (wjt) [inline gnome-help/C/shell-apps-open.page:49]: This isn't quite true. Pressing Super triggers the activities overview. Double-pressing Super, or pressing Super+A, goes to the desktop.

File: gnome-help/C/shell-exit.page [hunk 17]
```diff
@@ -97,7 +108,7 @@
   password to log back in. If you don’t lock your screen, it will lock
   automatically after a certain amount of time.</p>
 
-  <p>To lock your screen, click the system menu on the right side of the top
+  <p>To lock your screen, click the User menu on the right side of the top
```
Comment (wjt) [inline gnome-help/C/shell-exit.page:111]: We should drop this change: we used to replace GNOME's system menu with a user menu, but we have restored GNOME's system menu.

File: gnome-help/C/shell-introduction.page [hunk 18]
```diff
@@ -32,28 +33,42 @@
...
 
-  <p>GNOME features a user interface designed to stay out of your way, minimize
-  distractions, and help you get things done. When you first log in, you will
-  see the <gui>Activities</gui> overview and the top bar.</p>
+  <if:choose>
+
+    <if:when test="platform:endless">
+      <note style="tip">
+        <p>Endless OS is a Linux based operating system which provides a simplified and 
+        streamlined user experience using a customized <gui>desktop</gui> environment forked from GNOME 4.</p>
```
Comment (wjt) [inline gnome-help/C/shell-introduction.page:54]: I think the point of this paragraph was to explain what the term "GNOME" means, but I've always been a bit unhappy with it. This is pretty much the first page of the documentation and we're using the term "Linux" and "forked". I think we should reword it to something like (improvements and suggestions welcome!):

```suggestion
        <p>The Endless OS desktop is a customized version of a desktop called “GNOME”. You will see the terms “Endless OS” and “GNOME” used interchangeably in this documentation.</p>
```

File: gnome-help/C/shell-introduction.page [hunk 19]
```diff
@@ -189,12 +229,11 @@ like "Application windows" by using if:when and if:else ? -->
 
 </section>
 
-
-<section id="systemmenu">
-  <title>System menu</title>
+<section id="usermenu">
```
Comment (wjt) [inline gnome-help/C/shell-introduction.page:232]: As above, we no longer have a user menu. We should stick with the upstream term "system menu" here and below.

File: gnome-help/C/shell-introduction.page [hunk 20]
```diff
@@ -276,8 +320,16 @@ messages will be presented, such as when your battery is critically low.</p>
     workspaces. To switch to a different workspace, select the workspace you
     want to use.</p>
   </if:when>
+  <if:when test="platform:endless">
+    <media type="image" src="figures/shell-window-list-endless.png" width="400" if:test="!target:mobile">
+      <p>Window List</p>
```
Comment (wjt) [inline gnome-help/C/shell-introduction.page:325]: I think we could refer to this as either the dash (following the existing  GNOME terminology and used elsewhere in this documentation) or the dock, rather than window list.

File: gnome-help/C/shell-keyboard-shortcuts.page [hunk 21]
```diff
@@ -37,7 +42,7 @@
 <title>Useful keyboard shortcuts</title>
 
 <p>This page provides an overview of keyboard shortcuts that can help you
-use your desktop and applications more efficiently. If you cannot use a
+use your <gui>desktop</gui> and applications more efficiently. If you cannot use a
```
Comment (wjt) [inline gnome-help/C/shell-keyboard-shortcuts.page:45]: As above, let's not invalidate all translations of this paragraph:

```suggestion
use your desktop and applications more efficiently. If you cannot use a
```

File: gnome-help/C/shell-keyboard-shortcuts.page [hunk 22]
```diff
@@ -48,7 +53,7 @@ information on navigating user interfaces with only the keyboard.</p>
       <keyseq><key>Alt</key><key>F1</key></keyseq> or the</p>
       <p><key xref="keyboard-key-super">Super</key> key
     </p></td>
-    <td><p>Switch between the <gui>Activities</gui> overview and desktop. In
+    <td><p>Switch between the <gui>Activities</gui> overview and <gui>desktop</gui>. In
```
Comment (wjt) [inline gnome-help/C/shell-keyboard-shortcuts.page:56]:
```suggestion
    <td><p>Switch between the <gui>Activities</gui> overview and desktop. In
```

File: gnome-help/C/shell-windows.page [hunk 23]
```diff
@@ -21,9 +26,8 @@
 
   <title>Windows and workspaces</title>
 
-  <p>Like other desktops, the system uses windows to display your running
-  applications. Using both the <gui xref="shell-introduction#activities">Activities</gui>
-  overview and the <em>dash</em>, you can launch new applications and control
+  <p>Like other <gui>desktop</gui>, the system uses windows to display your running
```
Comment (wjt) [inline gnome-help/C/shell-windows.page:29]:
```suggestion
  <p>Like other desktops, the system uses windows to display your running
```

File: gnome-help/C/user-add.page [hunk 24]
```diff
@@ -55,8 +59,8 @@
       type in your password when prompted.</p>
     </item>
     <item>
-      <p>Press the <gui style="button">+</gui> button, below the list of
-      accounts on the left, to add a new user account.</p>
+      <p>Press the <gui style="button">Add User...</gui> button, to the top 
+      right of accounts on the left, to add a new user account.</p>
```
Comment (wjt) [inline gnome-help/C/user-add.page:63]: Great catch! I suggest a simplification:

```suggestion
      <p>Press the <gui style="button">Add User...</gui> button, at the top of the window, to add a new user account.</p>
```

File: gnome-help/C/user-delete.page [hunk 25]
```diff
@@ -62,8 +66,8 @@
     </item>
     <item>
       <p>Select the user that you want to delete and press the
-      <gui style="button">-</gui> button, below the list of accounts on the
-      left, to delete that user account.</p>
+      <gui style="button">Remove User...</gui> button, above the list of accounts on the
```
Comment (wjt) [inline gnome-help/C/user-delete.page:69]: The button is not above the list of accounts. I suggest splitting this step in two:

- Select the user that you want to delete.
- Scroll to the bottom of the window and press the <gui style="button">Remove User...</gui> button to delete that user account.

**Review by wjt — COMMENTED** [review]
This is very thorough work! Looking pretty close.

**Review by cassidyjames — CHANGES_REQUESTED** [review]
Just a few things I noticed. It might not be worth the time, but I also wonder if any time we show `<gui>desktop</gui>` we should still link to the section explaining it, e.g. `<gui xref="shell-introduction#activities">desktop</gui>`.

**Review by cassidyjames — APPROVED** [review]
I think this is good to merge from my perspective; we can continue to review in context and iterate if needed.

@MichaelDrydenEndless I think you'll need to mark the PR as ready to merge instead of a draft.
